pex/jobs.py

```diff
@@ -230,14 +216,23 @@ def spawn_jobs():
 
     try:
       if isinstance(item, Exception):
-        error = item
+        if raise_type:
+          error = item
+        else:
+          # Otherwise, if we were given no `raise_type`, we skip over the input that raised.
```

Comment:
Check on understanding: how come we don't want to log when this case happens? If I'm reading correctly, this case will never result in the log line 236 triggering. 

If we're worried about too wordy of logging, consider setting it as debug.

Reply:
Just an oversight. Logging a spawn failure (OSError, IOError) seems just as valuable as logging Job.Error non-zero exits below. I'll fix this.

Reply:
Ah, right. I was probably being lazy as well. There is unuseful info here since we don't have the input that led to the exception, unlike the the case below. That said, That can be fixed too.